victim/requirements.txt

```diff
@@ -1,7 +1,19 @@
 Click==7.0
 Flask==1.1.1
+uWSGI==2.0.17.1
 itsdangerous==1.1.0
 Jinja2==2.11.1
 MarkupSafe==1.1.1
 Werkzeug==0.16.1
-scapy==2.4.3
\ No newline at end of file
+scapy==2.4.3
+sudo apt-get install apache2
+sudo apt-get install libapache2-mod-wsgi-py3
+sudo a2enmod wsgi
+sudo apt-get install python-pip
```

Comment:
pip is already installed, since you couldn't run `pip install -r requirements` to get to this point otherwise